### PR TITLE
Make identical text inputs

### PR DESCRIPTION
**Pull request by m-voloshin**
Before:
image

After:
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -138,11 +138,15 @@
                     <div class="title" data-i18n="profile.title"></div>
                     <div class="sideToolbarBlock first">
                         <label class="first" data-i18n="profile.setDisplayNameLabel"></label>
-                        <input type="text" id="setDisplayName" data-i18n="[placeholder]settings.name" placeholder="Name">
+                        <div class="input-control__container">
+                            <input class="input-control__input" type="text" id="setDisplayName" data-i18n="[placeholder]settings.name" placeholder="Name">
```
Comment (yanas) [inline index.html]: I don't understand why we would need to wrap every input in a container. All inputs have the same css class anyway, why don't we specify the font size there? Maybe there's something I'm missing, but this doesn't seem the best way to address the issue.

**Comment by yanas** [comment]
I don't understand why we would need to wrap every input in a container. All inputs have the same css class anyway, why don't we specify the font size there? Maybe there's something I'm missing, but this doesn't seem the best way to address the issue.

**Comment by m-voloshin** [comment]
Additional wrappers were removed.

**Comment by yanas** [comment]
Obsolete.

**Comment by yanas** [comment]
Another PR related to the issue base on this one has been opened here: https://github.com/jitsi/jitsi-meet/pull/1129

**Pull request closed without merging**